src/include/migraphx/eliminate_data_type.hpp

```diff
@@ -40,7 +40,8 @@ struct module;
  */
 struct MIGRAPHX_EXPORT eliminate_data_type
 {
-    std::set<shape::type_t> types;
+    std::set<shape::type_t> unsupported_types;
+    std::set<std::string> unsupported_fp8_ops;
```

Comment:
This should just be named `unsupported_ops` or just `op_name`. It should also be initialized to empty by default and come after `target_type`.

Reply:
Done.